docs/content/users/quickstart.md

```diff
@@ -175,16 +176,36 @@ ddev launch
     # use the one-time link (CTRL/CMD + Click) from the command below to edit your admin account details.
     ddev drush uli
     ddev launch
+    ```
 
-    # For Drupal 11:
-    # ddev config --project-type=drupal --php-version=8.3 --docroot=web --corepack-enable
-    # ddev composer create drupal/recommended-project:^11.x-dev
+=== "Drupal 11 (dev)"
+
+    ```bash
+    mkdir my-drupal-site
+    cd my-drupal-site
+    ddev config --project-type=drupal --php-version=8.3 --docroot=web --corepack-enable
+    ddev start
+    ddev composer create drupal/recommended-project:^11.x-dev
+    ddev composer require drush/drush
+    ddev drush site:install --account-name=admin --account-pass=admin -y
+    # use the one-time link (CTRL/CMD + Click) from the command below to edit your admin account details.
+    ddev drush uli
+    ddev launch
+    ```
 
+=== "Drupal 9 (EOL)"
 
-    Note that you need to make minor adjustments for obsolete Drupal versions.
-    For example, Drupal 9:
-    # ddev config --project-type=drupal --php-version=8.1 --docroot=web
-    # ddev composer create drupal/recommended-project:^9
+    ```bash
+    mkdir my-drupal-site
+    cd my-drupal-site
+    ddev config --project-type=drupal --php-version=8.1 --docroot=web
+    ddev start
+    ddev composer create drupal/recommended-project:^9
+    ddev composer require drush/drush
+    ddev drush site:install --account-name=admin --account-pass=admin -y
+    # use the one-time link (CTRL/CMD + Click) from the command below to edit your admin account details.
+    ddev drush uli
+    ddev launch
     ```
 
 === "Drupal 6/7"
```

Comment:
```suggestion
=== "Drupal 6 (EOL)/7"
```
A real good idea adding EOL to Drupal 9! would it make sense to clarify that for drupal 6 as well?

Reply:
IMO they're so long gone that people should know this themselves, and will be using them deliberately if needed.